docs/BR.md

```diff
@@ -452,6 +455,8 @@ The script outputs:
 
 **Root Certificate**: The self-signed Certificate issued by the Root CA to identify itself and to facilitate verification of Certificates issued to its Subordinate CAs.
 
+**Short-lived Subscriber Certificate**: A server Certificate with a Validity Period that is less than ten (10) days (864,000 seconds). 
```

Comment:
On #402, @BenWilson-Mozilla commented...

```I would add "that is" so that it reads, "with a Validity Period that is less than ..."```

This change has been incorporated.